### PR TITLE
Internationalization added to all modals and non kanban components

### PR DESCRIPTION
**Pull request by gitoak**
# Pull request

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:
1. [x] Did you lint your code locally before submission?
2. [x] Did you test your feature thoroughly to ensure there are no bugs?
3. [ ] Does your feature introduce breaking changes?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Please describe your changes
* What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
* Describe what your code exactly does and if it affects any other part of the code in any way (especially check for breaking changes)

### Additional context
- text was moved to locales file
- German translation

> ⚠️ **Atention:** Please review/check the app after this pr as i dont know where all off the modals apear. There may be some typos in the i18n calls wich i missed even though i checked thoroughly.

### CODE REVIEW

File: components/modal/RenameBoard.vue [hunk 1]
```diff
@@ -24,17 +24,17 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
...
           />
         </div>
         <section id="inputs" class="mt-4 flex flex-col">
-          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName"
-            >Board Name</label
-          >
+          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName">{{
+            $t("modals.renameBoard.name")
+          }}</label>
           <input
```
Comment (trobonox) [inline components/modal/RenameBoard.vue:38]: The placeholder is not translated

File: components/modal/NewBoard.vue [hunk 2]
```diff
@@ -24,17 +24,17 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
...
           />
         </div>
         <section id="inputs" class="mt-4 flex flex-col">
-          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName"
-            >Board Name</label
-          >
+          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName">{{
+            $t("modals.newBoard.name")
+          }}</label>
           <input
```
Comment (trobonox) [inline components/modal/NewBoard.vue:38]: Placeholder of the input needs a translation

**Review by trobonox — CHANGES_REQUESTED** [review]
Placeholders are missing translations

File: components/modal/NewBoard.vue (hunk 2 above)
Comment (gitoak) [reply components/modal/NewBoard.vue:38]: ok fixed now

**Review by trobonox — APPROVED** [review]

**Pull request merged**